docs/guides/setup-channel.mdx

```diff
@@ -113,6 +113,76 @@ result = job.result()
 print(result)
   ```
 
+<span id="iqp-rest-api"></span>
+## Set up to use IBM Quantum Platform with REST API
+
+Alternatively, you can also access quantum processors with REST APIs, enabling you to work with QPUs using any programming language or framework. 
```

Comment:
Curious if it's standard to use the plural "REST APIs" or just "REST API"?

Reply:
"REST APIs" is used to refer to APIs - Runtime and Transpiler, etc. 
"REST API" is used to refer to just a specific API - either Runtime or Transpiler.